cmake/ImportLLVM.cmake

```diff
@@ -39,6 +41,14 @@ Once the modules are included we verify the LLVM version is supported and set
 some additional compile definitions.
 #]=======================================================================]
 
+# Set a flag for whether we are in an LLVM Tree and return early.
+# This is done by checking for the LLVMCore target which does not exist
+# until we import LLVM.
+set(CA_IN_LLVM_TREE OFF)
+if (TARGET LLVMCore)
+  set(CA_IN_LLVM_TREE ON)
```

Comment:
Am I missing something - should this not `return()` here? I thought if this falls through in an in-tree build, the next line would `FATAL_ERROR` because `CA_LLVM_INSTALL_DIR` isn't set.

Reply:
moved as suggested, so now no need for the return(). not sure what happened to the return() but you are correct.